File: todo_list.py
tasks = []

while True:
    print("\n📝 To-Do List")
    print("1️⃣ Add Task")
    print("2️⃣ View Tasks")
    print("3️⃣ Exit")
    
    choice = input("Choose an option: ")
    
    if choice == "1":
        task = input("Enter a new task: ")
        tasks.append(task)
        print("✅ Task added!")
    elif choice == "2":
        print("\n📋 Your Tasks:")
        for i, task in enumerate(tasks, 1):
            print(f"{i}. {task}")
    elif choice == "3":
        print("👋 Goodbye!")
        break
    else:
        print("❌ Invalid choice, try again.")

File: ToDoList.java
import java.util.ArrayList;
import java.util.Scanner;

class ToDoList {
    public static void main(String[] args) {
        ArrayList<String> tasks = new ArrayList<>();
        Scanner sc = new Scanner(System.in);
        int choice;

        while (true) {
            System.out.println("\n📝 To-Do List Menu:\n1️⃣ Add Task\n2️⃣ View Tasks\n3️⃣ Remove Task\n4️⃣ Exit");
            System.out.print("Choose an option: ");
            choice = sc.nextInt();
            sc.nextLine(); // Consume newline

            if (choice == 1) {
                System.out.print("📝 Enter task: ");
                tasks.add(sc.nextLine());
            } else if (choice == 2) {
                System.out.println("📋 Tasks:");
                for (int i = 0; i < tasks.size(); i++) {
                    System.out.println((i + 1) + ". " + tasks.get(i));
                }
            } else if (choice == 3) {
                System.out.print("❌ Enter task number to remove: ");
                int index = sc.nextInt();
                if (index > 0 && index <= tasks.size()) {
                    tasks.remove(index - 1);
                    System.out.println("✅ Task removed.");
                } else {
                    System.out.println("❌ Invalid task number.");
                }
            } else if (choice == 4) {
                System.out.println("👋 Exiting...");
                break;
            } else {
                System.out.println("❌ Invalid option!");
            }
        }

        sc.close();
    }
}
